dwave/system/samplers/dwave_sampler.py

```diff
@@ -35,25 +35,15 @@
 class DWaveSampler(dimod.Sampler, dimod.Structured):
     """A class for using the D-Wave system as a sampler.
 
-    Inherits from :class:`dimod.Sampler` and :class:`dimod.Structured`.
-
-    Enables quick incorporation of the D-Wave system as a sampler in
-    the D-Wave Ocean software stack. Also enables optional customizing of input
-    parameters to :std:doc:`D-Wave Cloud Client <cloud-client:index>`
-    (the stack's communication-manager package).
+    Uses parameters set in a configuration file, as environment variables, or explicitly as input arguments for selecting and communicating with a D-Wave system. For more information, see
```

Comment:
80 char width please - I know the package is not consistent but trying to keep it at that length going forward.

Reply:
Done